### PR TITLE
use CI IASQL_ENV on upgrade test and dev docker

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

**Review by mtp1376 — COMMENTED** [review]
Should these envs be `ci`?

**Comment by depombo** [comment]
> Should these envs be `ci`?

all `ci` is the same config as `local` with a more verbose logger and no telemetry

**Review by depombo — COMMENTED** [review]
`ci` is the same config as `local` with a more verbose logger and no telemetry

**Review by dfellis — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -24,8 +24,8 @@
     "docker-build:dev": "docker build -f ./Dockerfile.dev -t iasql:latest-dev .",
     "docker-run": "docker run -p 9876:9876 -p 5432:5432 --name iasql iasql:latest",
     "docker-run:detach": "docker run -d -p 9876:9876 -p 5432:5432 --name iasql iasql:latest",
-    "docker-run:dev": "docker run -p 9876:9876 -p 5432:5432 -p 127.0.0.1:9229:9229 --name iasql-dev -v $PWD/src:/engine/src -v /engine/node_modules iasql:latest-dev",
-    "docker-run-vol": "docker run -p 9876:9876 -p 5432:5432 -v $(dirname $PWD)/db:/var/lib/postgresql/14/main --name iasql iasql:latest",
+    "docker-run:dev": "docker run -p 9876:9876 -p 5432:5432 -e IASQL_ENV=ci -p 127.0.0.1:9229:9229 --name iasql-dev -v $PWD/src:/engine/src -v /engine/node_modules iasql:latest-dev",
+    "docker-run-vol": "docker run -p 9876:9876 -p 5432:5432 -e IASQL_ENV=ci -v $(dirname $PWD)/db:/var/lib/postgresql/14/main --name iasql iasql:latest",
```
Comment (aguillenv) [inline package.json:28]: why this? to not mess the telemetry?

Comment (aguillenv) [reply package.json:28]: just read the comment above

**Review by aguillenv — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (depombo) [reply package.json:28]: yes, the upgrade test is sending telemetry to posthog

**Pull request merged**